tools/webpack

```diff
@@ -23,7 +27,11 @@ def build_for_dev_server(host: str, port: str, minify: bool, disable_host_check:
     # This is our most dynamic configuration, which we use for our
     # dev server.  The key piece here is that we identify changes to
     # files as devs make edits and serve new assets on the fly.
-    webpack_args = ["node", "node_modules/.bin/webpack-cli", "serve"]
+    webpack_args = ["node"]
+    with open("/proc/meminfo") as meminfo:
+        if int(next(meminfo).split()[1]) < 3 * 1024 * 1024:
+            webpack_args += ["--max-old-space-size=1536"]
+    webpack_args = ["node_modules/.bin/webpack-cli", "serve"]
```

Comment:
```suggestion
    webpack_args += ["node_modules/.bin/webpack-cli", "serve"]
```

Reply:
This turned out to be unnecessary for development mode anyway.